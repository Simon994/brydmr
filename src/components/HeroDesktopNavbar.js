import React from 'react'
import { NavLink, withRouter } from 'react-router-dom'

import Logo from '../styles/assets/Logo.png'

function HeroDesktopNavbar() {

  return (
    <nav className='desktop-nav'>
      <div className='left-container'>
        <ul>
          <li>STORE</li>
          <li>TUTORIALS</li>
        </ul>
      </div>
      <img 
        src={Logo}
        alt='ByrdMR logo'
        className='byrd-logo'
      />
      <div className='right-container'>
        <ul>
          <li>BLOG</li>
          <li>CONTACT</li>
        </ul>
      </div>
    </nav>
  )
}

export default HeroDesktopNavbar