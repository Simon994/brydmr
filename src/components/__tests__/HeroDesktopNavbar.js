import React from 'react'
import renderer from 'react-test-renderer'
import HeroDesktopNavbar from '../HeroDesktopNavbar'

describe('The navbar for hero view on desktop device', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<HeroDesktopNavbar/>).toJSON()

    console.log(tree)
    expect(tree).toMatchSnapshot()
  })
})
