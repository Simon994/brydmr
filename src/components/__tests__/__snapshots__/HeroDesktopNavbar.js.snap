// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The navbar for hero view on desktop device renders as expected 1`] = `
<nav
  className="desktop-nav"
>
  <div
    className="left-container"
  >
    <ul>
      <li>
        <a
          href="#"
        >
          STORE
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          exact to=
          /tutorials
          className='nav-link' activeClassName='selected' &gt;
          TUTORIALS
        </a>
      </li>
    </ul>
  </div>
  <img
    alt="ByrdMR logo"
    className="byrd-logo"
    src="Logo.png"
  />
  <div
    className="right-container"
  >
    <ul>
      <li>
        <a
          href="#"
        >
          BLOG
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          exact to=
          /contact
          className='nav-link' activeClassName='selected' &gt;
          CONTACT
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
